cartographer/mapping/internal/3d/local_trajectory_builder_3d.cc

```diff
@@ -302,13 +306,27 @@ LocalTrajectoryBuilder3D::AddAccumulatedRangeData(
         common::ToSeconds(sensor_duration.value());
     kLocalSlamInsertIntoSubmapFraction->Set(insert_into_submap_fraction);
   }
-  const auto accumulation_stop = std::chrono::steady_clock::now();
-  if (last_accumulation_stop_.has_value()) {
-    const auto accumulation_duration =
-        accumulation_stop - last_accumulation_stop_.value();
-    kLocalSlamLatencyMetric->Set(common::ToSeconds(accumulation_duration));
+  const auto wall_time = std::chrono::steady_clock::now();
```

Comment:
Can we reorder the code, so that we first do `wall_time`, the corresponding if block, update `last_wall_time_`. Then follow the pattern with CPU time`. Would be more readable.